src/ble/att/permission.rs

```diff
@@ -0,0 +1 @@
+pub struct AttPermission {}
```

Comment:
A new file for just 1 line seems a bit overeager. Maybe only split it up once `AttPermission` grows in functionality?

Reply:
I agree haha